src/functions/misc_screen.py

```diff
@@ -28,12 +28,14 @@ class MiscScreen(Adw.Bin):
     ipv_switch = Gtk.Template.Child()
     timeshift_switch = Gtk.Template.Child()
     zramd_switch = Gtk.Template.Child()
+    #theme_switch = Gtk.Template.Child()
```

Comment:
What's this? I thought if we would do any potential theming related options they would be in Moonstone?

Reply:
This is just leftover code from when we wanted to have it in jade, I'll remove it.

Reply:
Sounds good, it seems to be in `summary_screen.py` too